pkgs/applications/networking/instant-messengers/trillian-im/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchurl
+, pkgs
```

Comment:
Never, ever call the planet inside an expression. Pick each dependency you eventually need, one by one.

```suggestion
```